### ISSUE TITLE
File failed to load: /extensions/MathMenu.js and MathZoom.js

### ISSUE THREAD

**Issue by lerods** [rstudio/rmarkdown#1800]
Hi Everyone, 

I am working creating a dashboard like this.

![image](https://user-images.githubusercontent.com/52513599/79674363-154bcf00-81a8-11ea-92eb-dab8c68b0900.png)

It works when I generate the web page using Rstudio and the package flexdashboard [.](https://rmarkdown.rstudio.com/flexdashboard/) . But unfortunally when I save the html, and I try to open it using directly the browser it does not work.

It looks like this.

![image](https://user-images.githubusercontent.com/52513599/79674525-95beff80-81a9-11ea-895c-e00942cc9916.png)

It's empty¡¡

The html says (Bottom left).

File failed to load: /extensions/MathMenu.js
seconds later says:
File failed to load: /extensions/MathZoom.js

Could you please help me?

Thanks in advance

**Comment by cderv** [comment]
Hello, 

Can you provide an example Rmd code reproducing your issue ? It would help us help you. 
It is possible there is an issue somewhere with the specific content of your document. I see you use widgets for examples. 
Thank you.

> But unfortunally when I save the html

How to you "save" the html ? Rendering the document should produce a standalone html file by default. Is it the file you are using ? 

<details><summary>This small example should work fine</summary>

````r
---
title: "Storyboard Commentary"
output: 
  flexdashboard::flex_dashboard:
    storyboard: true
---

### Frame 1

```{r}
plot(iris)
```

*** 

Some commentary about Frame 1.

### Frame 2 {data-commentary-width=400}

```{r}
mtcars
```

*** 

Some commentary about Frame 2.
````
</details>

Thank you

**Comment by lerods** [comment]
Thanks for your help cderv.

Yes, I am using that standalone html file.

Sure. There is no problem, this is the code:

Thanks Again.

---
title: "Dashboard"
output: 
  flexdashboard::flex_dashboard:
    vertical_layout: fill

---

```{r setup, include=FALSE}
#LIBRERIAS USADAS
require("RPostgreSQL")
require("DBI")
library(tidyverse)
#library(plotly)
library(dplyr)
library(tidyr)
library(RColorBrewer)
library(lubridate)
library(ggplot2)

library(rmarkdown)
library(dygraphs)
library(xts)
library(flexdashboard)


```


```{r}

#Conexion  POSTGRES
drv <- dbDriver('PostgreSQL')  
db <- 'redexterna'  
host_db <- 'localhost'  
db_port <- '5433'  
db_user <- 'xxxx'  
db_password <- 'xxxx'

conn <- dbConnect(drv, dbname=db, host=host_db, port=db_port, user=db_user, password=db_password)

#importacion tabla
tx_fcs_redext <- dbGetQuery(conn, "select aliado, id_pos, id_pos_int, key_posid_fecha, recaudo_neto, tx_neto, fecha_tx_format
from dataanalisis.cargadatatxfcs_dia" )

#Colors
col <- c('#F93A1C','#F114C2', '#3B4FF0', '#77F03B')


#agrupacion de la info por la key fecha ID_POS, 
tx_posid_dia_agrupada <- tx_fcs_redext  %>%
  arrange(fecha_tx_format)%>%
  group_by(key_posid_fecha,id_pos, fecha_tx_format, aliado ) %>%
  summarise(recaudo = sum(recaudo_neto, na.rm = T),
            tx=sum(tx_neto, na.rm = T),
  ) 

```
Comportamiento General Red Externa: Transaccional y Recaudo {.storyboard}
=========================================


```{r}

```

-----------------------------------------------------------------------

### Cantidad de Dispostivos transaccionlamente activos.


```{r}


#PROCESO GENERAR TABLA POSID ACTIVOS X ALIADO X FECHA

Atimeseries_x_fecha_cnt_activos <- tx_posid_dia_agrupada %>%                
  group_by(aliado, fecha_tx_format) %>%        
  summarise(cantidad = n_distinct(id_pos )) 

Btimeseries_x_fecha_cnt_activos <- Atimeseries_x_fecha_cnt_activos%>% 
  pivot_wider(names_from = aliado, values_from = cantidad)

serie_activos_por_aliado = xts (x=Btimeseries_x_fecha_cnt_activos[,-1], order.by=Btimeseries_x_fecha_cnt_activos$fecha_tx_format)
#PROCESO GENERAR TABLA POSID ACTIVOS X ALIADO X FECHA


#PROCESO GENERAR GRAFICO POSID ACTIVOS X ALIADO X FECHA
dygraph(serie_activos_por_aliado ) %>% 
  dyRangeSelector() %>%
  dyAxis("y", label = "POSID Activos") %>%
  dyEvent("2020-03-20", "Inicio Aislamiento", labelLoc = "bottom")%>% 
  dyEvent("2020-04-05", "Inicio Semana Santa", labelLoc = "bottom")%>% 
  dyEvent("2020-04-12", "Fin Semana Santa", labelLoc = "bottom")%>% 
  dyGroup(c("GS", "MR", "PK", "SB"), drawPoints = TRUE, color = col)%>% 
  dyHighlight(highlightSeriesOpts = list(strokeWidth = 4))
#PROCESO GENERAR GRAFICO POSID ACTIVOS X ALIADO X FECHA

```

### Recaudo Red Externa por Aliado

Describe el comportamiento del recaudo por monto del aliado en el tiempo.

```{r}


#PROCESO GENERAR TABLA AGRUPACION RECAUDO X ALIADO X FECHA

Atimeseries_recaudo <- tx_posid_dia_agrupada %>%                
group_by(aliado, fecha_tx_format) %>%        
summarise(recaudo_agr = round((sum(recaudo, na.rm = T)/1000000), digits=2))


#es necesario hacer un pivot para dejarlo en la estructura requerida
Btimeseries_recaudo <- Atimeseries_recaudo%>% 
pivot_wider(names_from = aliado, values_from = recaudo_agr)


#xts permite convertir el dataframe en el formato que requiere dypgraph
serie_recaudo = xts (x=Btimeseries_recaudo[,-1], order.by=Btimeseries_recaudo$fecha_tx_format)
#PROCESO GENERAR TABLA AGRUPACION RECAUDO X ALIADO X FECHA

dygraph(serie_recaudo ) %>% 
  dyRangeSelector() %>%
  dyAxis("y", label = "Recaudo (Millones de Pesos)") %>%
  dyEvent("2020-03-20", "Inicio Aislamiento", labelLoc = "bottom")%>% 
  dyEvent("2020-04-05", "Inicio Semana Santa", labelLoc = "bottom")%>% 
  dyEvent("2020-04-12", "Fin Semana Santa", labelLoc = "bottom")%>% 
  dyGroup(c("GS", "MR", "PK", "SB"), drawPoints = TRUE, color = col)%>% 
  dyHighlight(highlightSeriesOpts = list(strokeWidth = 4))

```

### Comportamiento Transaccional Red Externa por Aliado

Describe el comportamiento de las transacciones del aliado en el tiempo.

```{r}


Atimeseries_tx <- tx_posid_dia_agrupada %>%                
  group_by(aliado, fecha_tx_format) %>%        
  summarise(tx_agr = round((sum(tx, na.rm = T)/1000), digits=2))

Btimeseries_tx<- Atimeseries_tx%>% 
  pivot_wider(names_from = aliado, values_from = tx_agr)

serie_tx = xts (x=Btimeseries_tx[,-1], order.by=Btimeseries_tx$fecha_tx_format)
#PROCESO GENERAR TABLA AGRUPACION RECAUDO X ALIADO X FECHA

dygraph(serie_tx) %>% 
  dyRangeSelector() %>%
  dyAxis("y", label = "Transacciones(Miles)") %>%
  dyEvent("2020-03-20", "Inicio Aislamiento", labelLoc = "bottom")%>% 
  dyEvent("2020-04-05", "Inicio Semana Santa", labelLoc = "bottom")%>% 
  dyEvent("2020-04-12", "Fin Semana Santa", labelLoc = "bottom")%>% 
  dyGroup(c("GS", "MR", "PK", "SB"), drawPoints = TRUE, color = col)%>% 
  dyHighlight(highlightSeriesOpts = list(strokeWidth = 4))


```

### Comportamiento Promedio de Recarga Red Externa por Aliado

```{r}


Atimeseries_promedio_recarga <- tx_posid_dia_agrupada %>%                
  group_by(aliado, fecha_tx_format) %>%        
  summarise(recaudo_agr = (sum(recaudo, na.rm = T)), tx_agr = (sum(tx, na.rm = T)))%>% 
mutate(promedio_recarga= round((recaudo_agr/tx_agr), digits=0))

Atimeseries_promedio_recarga <- select(Atimeseries_promedio_recarga, aliado, fecha_tx_format, promedio_recarga)


Btimeseries_promedio_recarga <- Atimeseries_promedio_recarga%>% 
  pivot_wider(names_from = aliado, values_from = promedio_recarga)

serie_promedio = xts (x=Btimeseries_promedio_recarga [,-1], order.by=Btimeseries_promedio_recarga $fecha_tx_format)

dygraph(serie_promedio ) %>% 
  dyRangeSelector() %>%
  dyAxis("y", label = "Promedio de Recarga") %>%
  dyEvent("2020-03-20", "Inicio Aislamiento", labelLoc = "bottom")%>% 
  dyEvent("2020-04-05", "Inicio Semana Santa", labelLoc = "bottom")%>% 
  dyEvent("2020-04-12", "Fin Semana Santa", labelLoc = "bottom")%>% 
  dyGroup(c("GS", "MR", "PK", "SB"), drawPoints = TRUE, color = col)%>% 
  dyHighlight(highlightSeriesOpts = list(strokeWidth = 4))


```

**Comment by cderv** [comment]
I can't reproduce using Dygraphs. 

After research on Google, MathMenu.js and MathZoom.js seems to be related to MathJax, and an error loading them can be related to a browser. 

Unless you changed it, the default mathjax used by rmarkdown is from online. When you open the doc in your browser it will load from the website. You should look in the developper tool windows in your browser (F12 to open), to look at the possible loading error message. 

Without being able to reproduce or running you doc my self, I can't easily help further... 

You can also try another browser or on another computer to see if your file can be open there or not. 🤷

**Comment by lerods** [comment]
Thanks a lot¡, 

I understand, no problem.
I did what you suggested. I found this

![image](https://user-images.githubusercontent.com/52513599/79700698-53ef9100-825d-11ea-9f1e-46d54f8f7b04.png)


But I tried to save the html standalone in a differente way, using this option:
**Webpage, html only**, and it worked¡. 

![image](https://user-images.githubusercontent.com/52513599/79700795-0e7f9380-825e-11ea-99a7-e4fb405f0841.png)

Before I was trying to save the html standalone using **webpage, complete** and in this way, the file did not work.

I am not an expert, but I thought the html should to work using any format html, I have no idea if this was a silly mistake. But I would like to share the solution.

Thanks for your time an help cderv¡
